textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/TestConstants.java

```diff
@@ -65,4 +65,52 @@
         return Arrays.asList(tuple1, tuple2, tuple3, tuple4, tuple5);
                 
     }
+    
+    //sample created by team3

```

Comment:
Agreed.
TestConstants.java is located in src/**main**/.../common/constants. But considering it contains data related to test, how about move it to src/**test**/.../common/constants?